var/spack/repos/builtin/packages/libabigail/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Libabigail(AutotoolsPackage):
+    """The ABI Generic Analysis and Instrumentation Library"""
+
+    homepage = "https://sourceware.org/libabigail"
+    url      = "https://mirrors.kernel.org/sourceware/libabigail/libabigail-1.8.tar.gz"
+
+    version('1.8', sha256='1cbf260b894ccafc61b2673ba30c020c3f67dbba9dfa88dca3935dff661d665c')
+
+    depends_on('doxygen', type='build')
```

Comment:
one last thing -- for most things that require doxygen, we make it optional (b/c it takes a while to build and in many cases people do not need the docs).  Can you make a variant for this, e.g.:

```python
variant("docs", default=False, description="build doxygen documentation")
depends_on("doxygen", type="build, when="+docs")
```

Reply:
Should we add py-sphinx to that?

Reply:
Based on reading here https://sourceware.org/git/?p=libabigail.git;a=blob;f=COMPILING;h=33613a3708b73f2dcac772300876a2d6648bb425;hb=HEAD I added py-sphinx too for the docs variant.